android/app/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+	android:layout_height="match_parent"
+	android:layout_width="match_parent"
+	android:orientation="vertical">
+
+	<TextView
+		android:layout_width="match_parent"
+		android:layout_height="wrap_content"
+		android:text="Log In"
+		android:textAlignment="center"
+		android:layout_marginTop="30dp"
+		android:textSize="40sp"/>
+
+	<TextView
+		android:layout_width="match_parent"
+		android:layout_height="wrap_content"
+		android:layout_marginHorizontal="10dp"
+		android:text="Email:"
+		android:textSize="20sp" />
+
+	<EditText
+		android:layout_width="match_parent"
+		android:layout_height="wrap_content"
+		android:inputType="textEmailAddress"
+		android:layout_marginHorizontal="10dp"
+		android:textSize="20sp"
+		android:id="@+id/username"
+		/>
+
+	<TextView
+		android:layout_width="match_parent"
+		android:layout_height="wrap_content"
+		android:layout_marginHorizontal="10dp"
+		android:text="Password:"
+		android:textSize="20sp" />
+
+	<EditText
+		android:layout_width="match_parent"
+		android:layout_height="wrap_content"
+		android:inputType="textPassword"
+		android:layout_marginHorizontal="10dp"
+		android:textSize="20sp"
+		android:id="@+id/password"
+		/>
+
+	<Button

```

Comment:
Also include a register button.

Reply:
I am planning on making a separate register activity since it requires the name field as well. I will add the register button on the login activity, but it will redirect the user to the "create account" activity. I will also make the create account activity the default activity after I create it.

Reply:
I'd say that login should be the default activity if not signed in because people could be coming from the website/android.